File: app/src/main/java/com/arsbars/reminderandroid/LoginActivity.java
package com.arsbars.reminderandroid;

import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.arsbars.reminderandroid.view.fragments.LoginFragment;

public class LoginActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);
        InitiateView();
    }

    private void InitiateView() {
        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
        fragmentTransaction.replace(R.id.loginRootLayout, LoginFragment.newInstance());
        fragmentTransaction.commit();
    }
}

File: app/src/main/java/com/arsbars/reminderandroid/LoginActivity.kt
package com.arsbars.reminderandroid

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.constraint.ConstraintLayout
import com.arsbars.reminderandroid.view.fragments.LoginFragment

class LoginActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)

        val fragmentManager = supportFragmentManager
        val fragmentTransaction = fragmentManager.beginTransaction()

        val rootLayout = findViewById<ConstraintLayout>(R.id.loginActivityRootLayout)
        if (rootLayout.childCount == 0) {
            fragmentTransaction.replace(R.id.loginActivityRootLayout,
                    LoginFragment.newInstance())
        }
        fragmentTransaction.commit()
    }
}
